### ISSUE TITLE
Support "Show Items from subcollections"

### ISSUE THREAD

**Issue by tnajdek** [zotero/web-library#455]
Probably worth adding?

https://forums.zotero.org/discussion/92697/how-to-show-items-from-subcollection-in-web-library

**Comment by dstillman** [comment]
Would this need API support?

**Comment by tnajdek** [comment]
Come to think about it, yes it would.

**Comment by dstillman** [comment]
So this would need something like `collections/ABCD2345/items?includeSubcollections=1`?

**Comment by tnajdek** [comment]
Yes, I think that's the best way to approach this, if we want to support it. Also we would need tags-within-items endpoint, something like `collections/ABCD2345/items/[top/]tags`
